portal/import_template.php

```diff
@@ -508,11 +508,15 @@ function renderProfileHtml()
                             return true
                         },
                     },
+                    onAdd: function (evt) {
+                        let el = evt.item;
+                        el.getElementsByTagName('form')[0].classList.remove("d-none");
+                    },
                     animation: 150
                 });
             });
         });
-
+        top.restoreSession();
```

Comment:
above line looks out of place

Reply:
Nope was trying debugging session time out if the dialog stays open too long.
Restore session runs every time the script is loaded or that was the theory